bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_utils.py

```diff
@@ -70,7 +70,11 @@ def _string_to_root_attribute(value, lookup_map):
             return getattr(ROOT, lookup_map[value])
         else:
             try:
-                return getattr(ROOT, value)
+                # Here getattr(ROOT, value) would have less overhead, but it's
+                # also less convenient. With `eval`, the string that is passed
+                # by the user can also contain postfix operations on the enum
+                # value, which is often used for columns, e.g. `kGreen+1`.
+                return eval("ROOT." + value)
```

Comment:
Do we need a test of this new feature e.g. `pdf.plotOn(frame, LineColor="kOrange+1")` ?

Reply:
It's tested in the tutorials

Reply:
Ok! I have added a new unit test file with this (see the first commit in the PR, which got amended)